addons/edi_stock/data/edi_move_cancellation_data.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+        <!-- Move Cancellation sequence -->
+        <record id="sequence_edi_move_cancellation" model="ir.sequence">
+            <field name="name">Move Cancellation</field>
+            <field name="prefix">MCAN</field>
+            <field name="padding" eval="5" />
+        </record>
+        <!-- Move Cancellation record type -->
+        <record id="edi_record_type_move_cancellation" model="edi.record.type">
+            <field name="name">Move Cancellation</field>
+            <field name="sequence">67</field>
+            <field name="model_id" ref="model_edi_record_stock_cancellation" />
+        </record>
+        <!-- Move Cancellation document type -->
+        <record id="edi_document_type_move_cancellation" model="edi.document.type">
+            <field name="name">Move Cancellation</field>
+            <field name="model_id" ref="model_edi_document_stock_cancellation" />
+            <field name="sequence">23</field>
+            <field name="sequence_id" ref="sequence_edi_move_cancellation" />
+            <field name="rec_type_ids" eval="[(6, False, [
+          ref('edi_record_type_move_cancellation'),
+        ])]" />
+            <!-- pick_type_ids will be determined based on which ones have
+            u_enable_pick_cancellation_email_notifs set -->
+        </record>
+    </data>
+
+    <data noupdate="1">
+        <!-- Intended as a baseline template for Move Cancellation EDI, in a noupdate
+        block so that recipients can be configured without being overridden on upgrade -->
```

Comment:
Just check here that we can still set some of these in a customer module (if that's intended). Odoo can be funny with noupdates, sometimes setting it in one place work across all places we configure the specific record.

Reply:
Yeah, usually I would just remove the noupdate and add an xml inherit to update it for a customer.. However - for this specific story it's been requested that:
```There's an amendable list for email addresses to receive this update```
I'll double check how it behaves without a noupdate and with manually added recipients, as because we don't specify the field here it may persist

Reply:
tested and works fine without noupdate="1", so i'll remove